sass/custom/_component_page.scss

```
@@ -12,6 +12,35 @@
       border-color: #7c7c7c #c3c3c3 #ddd;
     }
   }
+
+  .integration_filter select {
+    width: 100%;
+    margin-bottom: 10px;
```

```suggestion
    margin-bottom: 15px;
```

With a single row this can be 15 instead, 16 maybe?

---

updated to 15px